README.md

```diff
@@ -39,9 +43,10 @@ and was compiled with Solidity 0.6.12.
 
 1. Create a file called `.env` with the following contents:
    ```bash
-   INFURA_ID=yourInfuraId
-   MNEMONIC_RINKEBY="your mnemonic for development" # only needed for deployment
+   MNEMONIC_TESTNET="your mnemonic for development" # only needed for deployment
    MNEMONIC_MAINNET="your mnemonic for production" # only needed for deployment
+   PROVIDER_URL_TESTNET="your provider url for development" # only needed for deployment
+   PROVIDER_URL_MAINNET="your provider url for production" # only needed for deployment
```

Comment:
Actually since this a one time deployment per network
Maybe we set the set RPC URL and just ask the user to passs the infura/equivalent ID ? / aka how it was